mw-webapp/src/styles/_variables.scss

```diff
@@ -1,13 +1,26 @@
+//Font family
+$primaryFontFamily: Inter;
+
 // Font size
-$font-size-large: 48px;
+$primaryFontSize: 12px;
+$fontSizeLarge: 48px;
 
 // Colors
 $shadow-color: rgb(211, 211, 211);
-$primary-background: rgb(255, 255, 255);
+$primaryBackgroundColor: rgb(255, 255, 255);
 $transparent-black: rgba(0, 0, 0, 90%);
 $borderBlack: rgb(0, 0, 0);
-$blackBackground: rgb(0, 0, 0);
-$whiteText: rgb(255, 255, 255);
+$secondaryBackgroundColor: rgb(0, 0, 0);
+$primaryTextColor: rgb(255, 255, 255);
+$primaryColor: rgb(99, 99, 99);
```

Comment:
please check where we use primaryColor. Let's get rid of it and use primaryBacjgroundColor or PrimaryTextColor or other similar

Reply:
`$shadow-color: rgb(211, 211, 211);

$primaryBackgroundColor: rgb(255, 255, 255);
$transparent-black: rgba(0, 0, 0, 90%);
$borderBlack: rgb(0, 0, 0);
$secondaryBackgroundColor: rgb(0, 0, 0);
$primaryTextColor: rgb(255, 255, 255);
$primaryColor: rgb(99, 99, 99);
$hoverColor: rgb(135, 135, 135);
$activeColor: rgb(45, 44, 44);`

Here primaryColor is the background color of the button according to styles in figma. We already have two background colors: primaryBackgroundColor and secondaryBackgroundColor. I can change the name primaryColor to tertiaryBackgroundColor but I can't change the color because it's defined in figma. Also I can't change the name primaryColor to primaryButtonColor because checkbox has the same color.
I was guided by the following articles in choosing the names: https://medium.com/@stefanmorris/proper-naming-of-color-variables-683264fec0f4 and https://joshuatz.com/posts/2021/common-css-color-variables-properties-theming/